supported_build_configurations.yaml

```diff
@@ -76,3 +76,6 @@ test-pwmin:
 #  - gamepad
 #  - gimbtonomy
 #  - science
+
+test-stress-can:
+  - science
```

Comment:
any particular reason for science board?

Reply:
its a board that i have